### ISSUE TITLE
If and else have different indents

### ISSUE THREAD

**Issue by sidred** [rust-lang-nursery/rustfmt#417]
```
let exact = diff / (if size == 0 {1} else {size});
```

becomes
```
let exact = diff /
            (if size == 0 {
    1
} else {
    size
});

```

In this case shouldn't the one line if else block be used? Or at least the else block should match the if block like this

```
let exact = diff /
            (if size == 0 {
                  1
              } else {
                  size
              });
```

Found in rust source src/libcollections/vec.rs line 1560.

**Comment by nrc** [comment]
Hmm, we usually use block indent here and it looks OK, I think this looks weird because the if/else is small and nested in parens. I wonder what we can do here.

**Comment by nrc** [comment]
Closed by #1017
